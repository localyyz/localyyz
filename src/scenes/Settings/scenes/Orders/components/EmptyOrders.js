import React from "react";
import { View, StyleSheet, Text } from "react-native";

// custom
import Support from "~/src/components/Support";
import { Styles, Sizes } from "localyyz/constants";

export default class EmptyOrders extends React.Component {
  render() {
    return (
      <Support title="Need help?">
        <View style={styles.container}>
          <Text style={styles.label}>
            You have no previously completed orders
          </Text>
        </View>
      </Support>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    marginHorizontal: Sizes.InnerFrame
  },

  label: {
    ...Styles.Text,
    textAlign: "center"
  }
});
