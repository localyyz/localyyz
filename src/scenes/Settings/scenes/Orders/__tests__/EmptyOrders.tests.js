import React from "react";
import Renderer from "react-test-renderer";
import EmptyOrders from "../components/EmptyOrders";

describe("EmptyOrders", () => {
  it("EmptyOrders: should render without failure", () => {
    const rendered = Renderer.create(<EmptyOrders />);
    expect(rendered.getInstance()).toBeDefined();
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
