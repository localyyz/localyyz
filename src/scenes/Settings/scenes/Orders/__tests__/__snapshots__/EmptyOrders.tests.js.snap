// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyOrders EmptyOrders: should render without failure 1`] = `
<View
  style={
    Object {
      "marginHorizontal": 15,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "backgroundColor": "rgba(255, 255, 255, 0)",
        "color": "rgb(30, 30, 30)",
        "fontSize": 16,
        "fontWeight": "300",
      }
    }
  >
    You have no previously completed orders
  </Text>
</View>
`;
